docs/api-reference/blocks/update-block.mdx

```diff
@@ -4,6 +4,13 @@ api: "PUT /api/pipelines/:pipeline_uuid/blocks/:block_uuid"
 ---
 `PUT /api/pipelines/:pipeline_uuid/blocks/:block_uuid`
 
+<ParamField query="name" type="string" required>
+  Human readable block name. This is used to generate the UUID, e.g. `restless dust`. Names will be slugified to a safe-uuid.
+</ParamField>
+<ParamField query="type" type="string" required>
+  Type of block:
+  `callback`, `chart`, `custom`, `data_exporter`, `data_loader`, `extension`, `dbt`, `scratchpad`, `sensor`, `transformer`
```

Comment:
Still looks like this list is missing `conditional`, `markdown`, and `global_data_product`